docs/setup_walkthrough.md

```diff
@@ -44,7 +44,9 @@ az pipelines variable-group variable create --group-id $vargroup_id --name servi
 az pipelines variable-group variable create --group-id $vargroup_id --name vargroup_id --secret false --value $vargroup_id &&
 az pipelines variable-group variable create --group-id $vargroup_id --name organization_name --secret false --value "$organization_name" &&
 az pipelines variable-group variable create --group-id $vargroup_id --name project_name --secret false --value "$project_name" &&
-az pipelines variable-group variable create --group-id $vargroup_id --name subscription_name --secret false --value "$subscription_name"
+az pipelines variable-group variable create --group-id $vargroup_id --name subscription_name --secret false --value "$subscription_name" &&
+az pipelines variable-group variable create --group-id $vargroup_id --name model_name --secret false --value "yolov5" &&

```

Comment:
Where do these new variables get used?

Reply:
They will be used in a new feature which is about to be added